website/docs/basics/global-settings.md

```diff
@@ -0,0 +1,121 @@
+---
+id: global-settings
+title: Global Settings
+sidebar_label: Global Settings
+---
+
+[![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs)
+
+Ok, this is the part that we are incredibly proud of. Did you know that you can pass values from the JSON manifest or some other file directly to SCSS? Yes, I had the same facial expression when I found out. For this, to work, we used Webpack and its excellent features.
+
+> This feature comes only with the block setup.
+
+So if you used the `init_theme` setup, you are all set; if not, please visit [blocks chapter](blocks) for more details.
+
+## Why we use this?
+
+Because with dynamic blocks, you must create your blocks, we got tired of copy/pasting styles from JavaScript/PHP/SCSS files. That is why we created this feature to define some global styles at one spot and use them in all three technologies.
+
+## How this works?
+
+You have file `src/Blocks/manifest.json` that has a key called `globalVariables`. We use this key and build the SCSS maps that you can use in your style files.
+
+Mind-blowing right?
+
+To use it, write your style in the `manifest.json` under the `globalVariables` key, and you can use that variable in your style files like this:
+
+```js
```

Comment:
```suggestion
```scss
```

Reply:
I will ignore all the comments like this because scss styling in not available in docusaurus and PHP style looks a blah gray and not good at all. So let's leave this like it is

Reply:
That's sad :(

Did you try adding additional languages: https://v2.docusaurus.io/docs/markdown-features/#syntax-highlighting

Odd that scss wouldn't be supported 🤷‍♂️

Reply:
I will check but not sure